src/pandablocks_ioc/ioc.py

```diff
@@ -1824,6 +1865,40 @@ def create_block_records(
 
         return record_dict
 
+    def create_version_records(self, fw_vers_dict: dict[EpicsName, str]):
```

Comment:
Big preference for unshortened variable names which don't reference type in method definitions.

```suggestion
    def create_version_records(self, firmware_versions: dict[EpicsName, str]):
```